projects/rero/ng-core/src/lib/record/search/record-search.component.html

```diff
@@ -56,9 +56,9 @@ <h5 *ngIf="typesInTabs.length === 1 && showLabel">
           <strong>{{ resultsText }}</strong>
         </div>
         <div class="col ml-auto text-right">
-          <a id="search-add-button" class="btn btn-primary" routerLink="new" *ngIf="adminMode.can && addStatus.can">
+          <a class="btn btn-primary" [routerLink]="addStatus?.url || ['new']" *ngIf="adminMode.can && addStatus.can">
             <i class="fa fa-plus mr-0 mr-sm-1"></i>
-            <span class="d-none d-sm-inline">{{ 'Add' | translate }}</span>
+            <span class="d-none d-sm-inline" translate>Add</span>
```

Comment:
Does it have to be in this PR?

Reply:
Not really... but why not :-)

Reply:
Why not but in this case please add an entry in the commit message.